edx_solutions_api_integration/management/commands/send_progress_leaderboard_notifications.py

```diff
@@ -47,16 +49,13 @@ def handle(self, *args, **options):
             aggregation_name='course',
             last_modified__gte=time_range
         ).distinct().values_list('course_key', flat=True)
-
         for course_key in courses:
             all_progress = Aggregator.objects.filter(
```

Comment:
Please add the join on this query instead, other wise we will miss positions for the staff/TA/observers. like if staff is on second no one will receive notification for second position.

Reply:
@nasirhjafri , I have updated the query